normalizer/modules/cowrie_events.py

```diff
@@ -0,0 +1,59 @@
+# Copyright (C) 2012 Johnny Vestergaard <jkv@unixcluster.dk>
```

Comment:
Yes, it's. For the same motivation specified in MHN repo PR, I prefer to have separated code which manage 2 different honeypots. Also because for now they share the same code, but in the future, with new additions to cowrie, the things can be different.